src/index.test.ts

```diff
@@ -432,3 +432,142 @@ test("get all icon explicitly", async () => {
     }"
   `)
 })
+
+test("test collection alias", async () => {
+  const result = await postcss([
+    tailwindcss({
+      config: {
+        content: [
+          {
+            raw: '<span class="i-heroiconsstroke-archive-box"></span>',
+            extension: "html",
+          },
+        ],
+        plugins: [
+          iconsPlugin({
+            collections: getIconCollections("all"),
+            collectionAliases: {
+              heroiconsstroke: {
+                collection: "heroicons",
+                extraProperties: {
+                  stroke: "#ffffff",
+                  "stroke-width": "2px",
```

Comment:
It would be better to switch to other more meaningful parameters, such as using the already provided `scale` or `strokeWidth`